sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/samples/CreateNewSampleSteps.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2021 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.sormas.e2etests.steps.web.application.samples;
+
+import static org.sormas.e2etests.pages.application.samples.CreateNewSamplePage.*;
+import static org.sormas.e2etests.pages.application.samples.SampleManagementPage.*;
+
+import com.google.common.truth.Truth;
+import cucumber.api.java8.En;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import javax.inject.Inject;
+import org.sormas.e2etests.helpers.WebDriverHelpers;
+import org.sormas.e2etests.pojo.Sample;
+import org.sormas.e2etests.services.SampleService;
+
+public class CreateNewSampleSteps implements En {
+  public static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("M/dd/yyyy");
+  public static final DateTimeFormatter TIME_FORMATTER = DateTimeFormatter.ofPattern("HH:mm");
+  public static Sample sample;
+  public static String sampleId;
+  private final WebDriverHelpers webDriverHelpers;
+
+  @Inject
+  public CreateNewSampleSteps(WebDriverHelpers webDriverHelpers, SampleService sampleService) {
+    this.webDriverHelpers = webDriverHelpers;
+
+    When(
+        "^I create a new Sample with specific data$",
+        () -> {
+          sample = sampleService.buildGeneratedSample();
+          webDriverHelpers.clickOnWebElementBySelector(INTERNAL_IN_HOUSE_TESTING);
```

Comment:
here we can change with selecting the option based on the String set in POJO

Reply:
agree

Reply:
We could but we would complicate it and would also raise the same expect to treat the rest of the buttons like Discard and Save the same way, when a simple click will do it.